docs/try-it-in-the-playground.md

```diff
@@ -0,0 +1,147 @@
+# Try it in the API Playground!
+
+The SDK ships with a command-line based API Playground. You can use
+the Playground to try out the SDK with real API access to learn and
+test things. It also supports executing scripts against the API.
+
+To start the Playground, go to the directory where you cloned the SDK
+Git repository and type:
+
+```
+$ yarn run playground --clientid <CLIENT-ID>
+```
+
+or use the shorthand:
+```
+$ yarn run pg -c <CLIENT-ID>
+```
+
+This will start the playground using your Marketplace API Client ID to
+connect to to your marketplace. You can create a new or find your
+existing Client ID in Flex Console at
+https://flex-console.sharetribe.com/applications.
+
+## Making API Requests
+
+You can make API requests to the Marketplace API and print the results
+using the built-in response printer:
+
+```js
+sdk.listings.query({per_page: 5}).then(printResponse);
+```
+
+You can also do custom processing of responses:
+
+```js
+sdk.listings.query({per_page: 10}).then(response => {
+  console.log("Fetched " + response.data.data.length + " listings.");
+  response.data.data.forEach(listing => {
+    console.log(listing.attributes.title);
+  });
+});
+```
+
+## Authenticated access
+
+To start the Playground in a user authenticated mode you can pass in
+the user info you want to log in with:
+
+```
+$ yarn run pg -c <CLIENT-ID> -u useremail@yourdomain.com -s user-secret-password
```

Comment:
Comment: Encouraging users to type passwords to command-line commands 😬

Reply:
The assumption is that playground is something you mostly run against your test environment. You don't even know the passwords of your real users unless you asked. It could be prompted to remedy this but that kills the history usage in command line which I think is how you mostly launch this. So I believe I'll stick with this version for now.